README.md

```diff
@@ -227,6 +227,45 @@ jobs:
           workdir: example/
 ```
 
+## Artifacts
+
+Each call to this action produces one metadata artifact as well as either one or two tarballs
+containing a tagged single-arch docker image.
+
+### Metadata: Tag List
+
+The metadata file is a JSON blob containing three lists of tags generated by the call to the action.
+The file is named either `docker_tag_list_${TARGET}.json` or `docker_tag_list_${TARGET}_redhat.json`.
+
+If you're defining `redhat_tag` then the filename will have the `_redhat` suffix.
+
+This tag list is used by downstream processes to quickly determine which tags were generated by
+the build, so they know which remote registries they'll require credentials for when uploading.
+
+Note that when calling this action in a matrix, each call will produce the same tag list file,
+because its name and contents are not dependent on the os and/or architecture being built.
+
+### Image Tarballs
+
+The image tarballs are generated by calls to `docker save`. Each one contains the full single-arch
+docker image, as well as all of the tags associated with that image. You will see that every image
+has a special tag in addition to those defined by the action's inputs. This is called the "auto tag"
+and its format uniquely identifies the single-arch image.
+
+### Auto Tag
+
+The "auto tag" is a semantic tag that identifies a single-arch image. It is used by downstream
+processes that group sets of single-arch images into multi-arch manifests.
+
+In order to create a multi-arch manifest from a set of single-arch images, downstream must call
+`docker load` for each single-arch image tarball. Because the user-defined tags are the same for
+each single-arch image, they cannot be used to differentiate the images loaded. Each `docker load`
+call points all the user-defined tags to the new single-arch image. However, the auto-tag, being
+different for each image, is retained after each `docker load` and can be used to identify which
+images need to be stitched together using `docker manifest create`.
+
+### Image
+
```

Comment:
Was there supposed to be content under this subheading?

Reply:
I think this should be removed, I'll do that.